scss/_blockquotes.scss

```diff
@@ -1,14 +1,6 @@
 @import 'o-quote/main';
 
 .n-content-blockquote {
-	@include oQuote();
-	@include oQuoteStandard();
-	@include oTypographyMargin($bottom: 7);
-
-	cite {
-		@include oQuoteCite();
-		@include oQuoteStandardCite();
-		@include oQuoteStandardCiteAuthor();
-	}
-
+	@include oQuote($opts: (cite: true, standard: true));
```

Comment:
i think for this component, we'd like to use the other placeholder mixins. Sorry, this isn't very well documented and that's my fault.

for this i'd

```
.n-content-blockquote {
	@include oQuoteBase();
	@include oQuoteStandard();

	cite {
		@include oQuoteBaseCite();
		@include oQuoteStandardCite();
		@include oQuoteStandardCiteAuthor();
	}

	margin-bottom: oSpacingByName('s8');
}
```